spec/controllers/admin/account_moderation_notes_controller_spec.rb

```diff
@@ -29,6 +29,7 @@
 
       it 'fails to create a note' do
         expect { subject }.to_not change(AccountModerationNote, :count)
+        expect(assigns(:moderation_notes).to_a).to eq []
```

Comment:
We should avoid adding *new usage* of `assigns` (see https://github.com/mastodon/mastodon/pull/30195)

Instead, either assert the presence/order of expected things on the page, or if the request would blow up completely when the i-var is nil, it's fine to just let the success response assertion sort of be a stand-in for "everything was set up"

Reply:
I mostly want to assert that `moderation_notes` is using the scope:

```
@moderation_notes = @account.targeted_moderation_notes.latest
```

Because that's duplicated between two controllers (reports_controller and report_notes_controller), it's important that the sort order is consistent between them

Reply:
I don't really understand how that assertion helps with asserting the order.

Reply:
I guess I can drop it. I just wanted to make sure when we render that view we definitely had stuff right.

I'm switching to using a default scope now, which I think will be better.

Reply:
Oh, @mjankowski on `assigns` I do agree with you, but perhaps we can tackle that later when we refactor the controller specs? Then we can assert that the page rendered a specific way?

Reply:
Yeah - fine either way. If it's easy to avoid here and detect on page instead, go ahead ... if not I'll wind up getting to this one eventually in my controller spec adventures.

Reply:
For now it's not easy to detect on page, but once we land #31529 it'll be easier, since each moderation note will have a unique ID on the page, so we can just assert each is incrementing or something.